src/test/e2e/favor-permission-name-over-id.e2e.test.ts

```diff
@@ -110,6 +110,9 @@ test('Favor permission name over id migration correctly assigns permissions by n
         { role_id: 101, permission: 'TEST_PERMISSION_4' },
     ]);
 
+    // Run the toggle to flag update migration
+    await dbm.up('20240523113322-roles-toggle-to-flag-rename.js');
```

Comment:
This test is manually applying migrations and it was previously running old names. Now I apply the new naming migration to clean it up.